package.json

```diff
@@ -25,7 +25,7 @@
     "bootstrap": "npm install lerna && lerna bootstrap && npm install",
     "format": "prettier --write \"packages/**/*.{ts,json}\"",
     "reset": "npm run cleanup && rimraf -rf node_modules && npm run bootstrap",
-    "build": "npm run cleanup && lerna run compile",
+    "build": "lerna run compile",
```

Comment:
@eamon0989 simply removing the global cleanup phase solved the issue of missing `dist`.
Since we are already cleanup build output individually for each package with `build` command, that should work fine.

Reply:
That's fantastic 👍